api/src/main/java/jakarta/servlet/http/HttpServlet.java

```diff
@@ -732,13 +767,19 @@ public void write(byte buf[], int offset, int len) throws IOException {
         contentLength += len;
     }
 
+    @Override
+    public void flush() throws IOException {
+        if (disableFlush.get() != Boolean.TRUE)
```

Comment:
This feels a bit yuck, because it is an identity comparison on Boolean.TRUE rather than the actual boolean comparison that it looks like. I know it does not really matter but maybe rewrite it to `Boolean.TRUE.equals(disableFlush.get())`?